pkg/run/cel.go

```diff
@@ -0,0 +1,306 @@
+package run
```

Comment:
Could you write unit tests for this file? It'd be sweet to see a bunch of small "this CEL string becomes this SQL string" tests

Reply:
sure, it's probably not gonna be much since the SQL translation only really happens for some portion of the event queries, and there are non for output.

Reply:
CEL -> SQL unit tests added.
It's still pretty basic since it's rather limited in usage.